base/common/unit.h

```diff
@@ -1,6 +1,10 @@
 #pragma once
 #include <cstddef>
 
+#if defined(__clang__) && __clang_major__ >= 13
```

Comment:
this fixes an error:
```
semin-serg@semin-serg-ub:~/ClickHouse/build-clang-13$ ninja
		[0/2] Re-checking globbed directories...
		[236/2018] Building CXX object src/CMakeFiles/dbms.dir/Disks/S3/DiskS3.cpp.o
		FAILED: src/CMakeFiles/dbms.dir/Disks/S3/DiskS3.cpp.o 
		/usr/bin/clang++-13  -DAWS_SDK_VERSION_MAJOR=1 -DAWS_SDK_VERSION_MINOR=7 -DAWS_SDK_VERSION_PATCH=231 -DBOOST_ASIO_STANDALONE=1 -DCARES_STATICLIB -DCONFIGDIR=\"\" -DENABLE_OPENSSL_ENCRYPTION -DHAVE_CONFIG_H -DLIBSASL_EXPORTS=1 -DLZ4_DISABLE_DEPRECATE_WARNINGS=1 -DOBSOLETE_CRAM_ATTR=1 -DOBSOLETE_DIGEST_ATTR=1 -DPLUGINDIR=\"\" -DPOCO_ENABLE_CPP11 -DPOCO_HAVE_FD_EPOLL -DPOCO_OS_FAMILY_UNIX -DSASLAUTHD_CONF_FILE_DEFAULT=\"\" -DSNAPPY_CODEC_AVAILABLE -DSTD_EXCEPTION_HAS_STACK_TRACE=1 -DUNALIGNED_OK -DUSE_JEMALLOC=1 -DUSE_REPLXX=1 -DUSE_XXHASH=1 -DWITH_COVERAGE=0 -DWITH_GZFILEOP -DX86_64 -DZLIB_COMPAT -D__CLANG_SUPPORT_DYN_ANNOTATION__ -I../contrib/sentry-native/include -Iincludes/configs -I../src -Isrc -Isrc/Core/include -I../base/glibc-compatibility/memcpy -I../base/common/.. -Ibase/common/.. -I../contrib/cityhash102/include -I../contrib/cctz/include -Icontrib/zlib-ng -I../contrib/zlib-ng -I../base/pcg-random/. -I../contrib/aws-c-common/include -I../contrib/aws-c-event-stream/include -Icontrib/aws-s3-cmake/include -Icontrib/cyrus-sasl-cmake -I../contrib/lz4/lib -I../base/mysqlxx/.. -I../contrib/mariadb-connector-c/include -Icontrib/mariadb-connector-c/include -Icontrib/grpc/third_party/cares/cares -I../contrib/grpc/third_party/cares/cares -I../contrib/abseil-cpp -I../contrib/libpq -I../contrib/libpq/include -I../contrib/libstemmer_c/include -I../contrib/wordnet-blast -I../contrib/lemmagen-c/include -isystem ../contrib/libpqxx/include -isystem ../contrib/rocksdb/include -isystem ../contrib/orc/c++/include -isystem contrib/orc/c++/include -isystem ../contrib/AMQP-CPP/include -isystem ../contrib/libuv/include -isystem ../contrib/AMQP-CPP -isystem ../contrib/s2geometry/src/s2 -isystem ../contrib/libhdfs3/include -isystem ../contrib/protobuf/src -isystem ../contrib/sparsehash-c11 -isystem ../contrib/openldap-cmake/linux_x86_64/include -isystem ../contrib/openldap/include -isystem ../contrib/avro/lang/c++/include -isystem ../contrib/miniselect/include -isystem ../contrib/pdqsort -isystem ../contrib/krb5/src/include -isystem contrib/krb5-cmake/include -isystem ../contrib/llvm/llvm/include -isystem contrib/llvm/llvm/include -isystem ../contrib/libcxx/include -isystem ../contrib/libcxxabi/include -isystem ../contrib/libunwind/include -isystem ../contrib/jemalloc-cmake/include -isystem ../contrib/libdivide/. -isystem ../contrib/croaring/cpp -isystem ../contrib/croaring/include -isystem ../contrib/fast_float/include -isystem ../contrib/msgpack-c/include -isystem ../contrib/aws/aws-cpp-sdk-s3/include -isystem ../contrib/aws/aws-cpp-sdk-core/include -isystem ../contrib/xz/src/liblzma/api -isystem ../contrib/zstd/lib -isystem ../contrib/re2 -isystem ../contrib/boost -isystem ../contrib/poco/Net/include -isystem ../contrib/poco/Foundation/include -isystem ../contrib/poco/NetSSL_OpenSSL/include -isystem ../contrib/poco/Crypto/include -isystem ../contrib/boringssl/include -isystem ../contrib/poco/Util/include -isystem ../contrib/poco/JSON/include -isystem ../contrib/poco/XML/include -isystem ../contrib/replxx/include -isystem ../contrib/fmtlib-cmake/../fmtlib/include -isystem ../contrib/double-conversion -isystem ../contrib/dragonbox/include -isystem contrib/re2_st -isystem ../contrib/cppkafka/include -isystem ../contrib/librdkafka-cmake/include -isystem ../contrib/librdkafka/src -isystem contrib/librdkafka-cmake/auxdir -isystem ../contrib/NuRaft/include -isystem ../contrib/poco/MongoDB/include -isystem ../contrib/icu/icu4c/source/i18n -isystem ../contrib/icu/icu4c/source/common -isystem ../contrib/capnproto/c++/src -isystem ../contrib/arrow/cpp/src -isystem ../contrib/arrow-cmake/cpp/src -isystem contrib/arrow-cmake/cpp/src -isystem ../contrib/avro/lang/c++/api -isystem ../contrib/grpc/include -isystem src/Server/grpc_protos -isystem ../contrib/s2geometry/src -isystem ../contrib/sqlite-amalgamation -isystem ../contrib/libc-headers/x86_64-linux-gnu -isystem ../contrib/libc-headers -fdiagnostics-color=always -fsized-deallocation  -gdwarf-aranges -pipe -mssse3 -msse4.1 -msse4.2 -mpclmul -mpopcnt -fasynchronous-unwind-tables -falign-functions=32   -Wall -Wno-unused-command-line-argument  -fdiagnostics-absolute-paths -fexperimental-new-pass-manager -Werror -Wextra -Wframe-larger-than=65536 -Wpedantic -Wno-vla-extension -Wno-zero-length-array -Wno-c11-extensions -Wcomma -Wconditional-uninitialized -Wcovered-switch-default -Wdeprecated -Wembedded-directive -Wempty-init-stmt -Wextra-semi-stmt -Wextra-semi -Wgnu-case-range -Winconsistent-missing-destructor-override -Wnewline-eof -Wold-style-cast -Wrange-loop-analysis -Wredundant-parens -Wreserved-id-macro -Wshadow-field -Wshadow-uncaptured-local -Wshadow -Wstring-plus-int -Wundef -Wunreachable-code-return -Wunreachable-code -Wunused-exception-parameter -Wunused-macros -Wunused-member-function -Wzero-as-null-pointer-constant -Weverything -Wno-c++98-compat-pedantic -Wno-c++98-compat -Wno-c99-extensions -Wno-conversion -Wno-ctad-maybe-unsupported -Wno-deprecated-dynamic-exception-spec -Wno-disabled-macro-expansion -Wno-documentation-unknown-command -Wno-double-promotion -Wno-exit-time-destructors -Wno-float-equal -Wno-global-constructors -Wno-missing-prototypes -Wno-missing-variable-declarations -Wno-nested-anon-types -Wno-packed -Wno-padded -Wno-shift-sign-overflow -Wno-sign-conversion -Wno-switch-enum -Wno-undefined-func-template -Wno-unused-template -Wno-vla -Wno-weak-template-vtables -Wno-weak-vtables -O2 -g -DNDEBUG -O3  -fno-pie   -D OS_LINUX -nostdinc++ -pthread -std=gnu++2a -MD -MT src/CMakeFiles/dbms.dir/Disks/S3/DiskS3.cpp.o -MF src/CMakeFiles/dbms.dir/Disks/S3/DiskS3.cpp.o.d -o src/CMakeFiles/dbms.dir/Disks/S3/DiskS3.cpp.o -c ../src/Disks/S3/DiskS3.cpp
		In file included from ../src/Disks/S3/DiskS3.cpp:12:
		/home/semin-serg/ClickHouse/base/common/unit.h:8:29: error: identifier '_KiB' is reserved because it starts with '_' followed by a capital letter [-Werror,-Wreserved-identifier]
		constexpr size_t operator"" _KiB(unsigned long long val) { return val * KiB; }
		                 ~~~~~~~~~~~^~~~
		                 operator""_KiB
		/home/semin-serg/ClickHouse/base/common/unit.h:9:29: error: identifier '_MiB' is reserved because it starts with '_' followed by a capital letter [-Werror,-Wreserved-identifier]
		constexpr size_t operator"" _MiB(unsigned long long val) { return val * MiB; }
		                 ~~~~~~~~~~~^~~~
		                 operator""_MiB
		/home/semin-serg/ClickHouse/base/common/unit.h:10:29: error: identifier '_GiB' is reserved because it starts with '_' followed by a capital letter [-Werror,-Wreserved-identifier]
		constexpr size_t operator"" _GiB(unsigned long long val) { return val * GiB; }
		                 ~~~~~~~~~~~^~~~
		                 operator""_GiB
		3 errors generated.
		[241/2018] Building CXX object src/CMakeFiles/dbms.dir/Functions/FunctionsLogical.cpp.o
		ninja: build stopped: subcommand failed.
```